crates/forge/tests/integration/meta_tx_v0.rs

```diff
@@ -1,12 +1,30 @@
 use forge_runner::forge_config::ForgeTrackedResource;
 use indoc::indoc;
+use scarb_api::ScarbCommand;
+use semver::Version;
 use std::path::Path;
 use test_utils::runner::{Contract, assert_passed};
 use test_utils::running_tests::run_test_case;
 use test_utils::test_case;
 
+// TODO(#3704) Remove scarb version check
+fn skip_if_scarb_lt_2_11_0() -> Option<String> {
+    match ScarbCommand::version().run() {
+        Ok(version_info) if version_info.scarb < Version::new(2, 11, 0) => {
+            Some("[IGNORED] `meta_tx_v0` syscall is not supported in Scarb < 2.11.0".to_string())
+        }
+        _ => None,
+    }
+}
+
 #[test]
 fn meta_tx_v0_with_cheat_caller_address() {
+    // TODO(#3704) Remove scarb version check
+    if let Some(skip_message) = skip_if_scarb_lt_2_11_0() {
+        eprintln!("{skip_message}");
+        return;
+    }
```

Comment:
Maybe let's keep the message printing in `skip_if_scarb_lt_2_11_0`, this way you could just do this


```suggestion
    skip_if_scarb_lt_2_11_0();
```